src/eslint/configuration.js

```diff
@@ -154,6 +154,23 @@ const typescriptRules = {
   "@typescript-eslint/no-empty-function": "off",
   "dot-notation": "off", // can conflict with "@typescript-eslint/dot-notation"
   "@typescript-eslint/dot-notation": "error",
+  camelcase: "off", // can conflict with @typescript-eslint/naming-convention
+  "@typescript-eslint/naming-convention": [
+    "error",
+    {
+      selector: "default",
+      format: ["camelCase"],
+      leadingUnderscore: "allow",
+      trailingUnderscore: "allow",
+    },
+    {
+      selector: "variable",
+      format: ["camelCase"],
+      leadingUnderscore: "allow",
+      trailingUnderscore: "allow",
+    },
```

Comment:
Shouldn't `selector: "default"` already match variables? 
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md#group-selectors

Reply:
True. Although I need to test this again because it didn't work as I expected.